src/style.css

```diff
@@ -0,0 +1,66 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body * {
+  font-family: "Avenir", sans-serif;
+  padding: 0;
+  margin: 0;
+  font-weight: 300;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-input {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  background: 25px no-repeat url(images/search.svg);
```

Comment:
You will need to rewrite it because on focus you should change background of input and for keep image you should type it like background-image property

Reply:
what about position small form?